Cargo.toml

```diff
@@ -172,6 +172,7 @@ sha3 = "=0.10"
 socket2 = "0.5"
 stream_limiter = "3.2"
 structopt = "0.3"
+clap = { version = "4.4.6", features = ["derive", "cargo"] }
```

Comment:
4.4 to comply to the general deps versioning

Reply:
Done :)